src/v/security/audit/schemas/utils.cc

```diff
@@ -227,6 +391,90 @@ std::ostream& operator<<(std::ostream& os, const ocsf_base_impl& impl) {
               << ", type_uid: " << impl.get_type_uid()() << "}";
 }
 
+event_type kafka_api_to_event_type(kafka::api_key key) {
```

Comment:
this can be `constexpr`

Reply:
I could if I moved the implementation to the header, but I didn't want to include all of the request headers in the `utils.h` header file, so I'm going to leave it as is for now.